src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -45,6 +56,8 @@ public class SwerveSubsystem extends SubsystemBase {
   /** Swerve drive object. */
   private final SwerveDrive swerveDrive;
 
+  private final LimelightSubsystem limelight;
```

Comment:
```suggestion
  private final LimelightSubsystem limelight = new LimelightSubsystem();
```

Remove the other initialization in the file.